package.json

```diff
@@ -10,7 +10,7 @@
   "license": "",
   "description": "Perf Widget front-end and back-end",
   "engines": {
-    "node": "5.x.x"
+    "node": "5.6.0"
```

Comment:
I'd agree, I don't think there's ever a really good reason for locking down to a specific version number, unless there are known incompatibilities.